src/render-html/js.js

```diff
@@ -204,7 +207,11 @@ const handleLongPress = e => {
 
   sendMessage({
     type: 'longPress',
-    target: e.target.matches('.header') ? 'header' : 'message',
+    target: e.target.matches('.header')
```

Comment:
Are we handling a thing pressed is empty space?
Not related to this PR but the code made me thing...

Reply:
Also, I am wondering if we should use the `node.classList.contains('header')` call?